agent/structs/connect_ca.go

```diff
@@ -365,6 +365,10 @@ var MaxLeafCertTTL = 365 * 24 * time.Hour
 // of the intermediate cert is checked and renewed if necessary.
 var IntermediateCertRenewInterval = time.Hour
 
+// NotBefore will be CertificateTimeDriftBuffer in the past to account for
+// time drift between different servers.
+var CertificateTimeDriftBuffer = time.Minute
```

Comment:
I think it might make sense to put this constant into `agent/connect` where it is used. Does that cause import problems?

Reply:
Good point! I put it into `agent/connect/ca` where it is being used and where it doesn't cause import problems.